.versionbot/CHANGELOG.yml

```diff
@@ -1,3 +1,14 @@
+- commits:
```

Comment:
These extra files may be just a matter of re-fetching the master branch and rebasing, no?

Reply:
I pulled clean into a fresh laptop... idk